app/api/protected/route.ts

```diff
@@ -1,6 +1,6 @@
 import { auth } from "auth"
 
-export const GET = auth((req) => {
+export const GET = auth ((req: any ) => {
```

Comment:
req is already typed why would you need any here?

![image](https://github.com/nextauthjs/next-auth-example/assets/24642451/1782fa70-2ad6-45af-99a2-201f949ab687)